app/assets/stylesheets/_base.scss

```diff
@@ -1558,6 +1568,10 @@ body {
     flex-grow: 1;
   }
 
+  .overview-flex-container > .flex-column:first-child {
+    padding-right: 32px;
```

Comment:
The `+` marker goes over text, so there was some overlap between the 2 columns where the location details are listed. I added the padding to the first column so make sure the overlap does not happen.
<img width="983" alt="Screenshot 2024-11-18 at 7 37 22 PM" src="https://github.com/user-attachments/assets/90fb80b0-7571-45c7-8d15-b93ba03d0781">